webserver/src/activity-logger/generator/reply/reply-generator.js

```
@@ -16,9 +16,10 @@ import { createCommentObject } from "../comment/comment-utils";
 
 let DOMAIN = "http://www.CourseMapper.de"; // TODO: Hardcoded due to frontend implementation
 
+//TODO: (Shoeb) The username should be anonymized
 const createUserObject = (req) => {
   let user = req.locals.user;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'user' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)